scrapers/scrape_bs.sh

```diff
@@ -1,6 +1,25 @@
 #!/usr/bin/env python3
+import requests
+from bs4 import BeautifulSoup
+
 
 import scrape_common as sc
+import utils.bs as bs
+
+def text_from_url(url: str) -> str:
+    req = requests.get(url)
+    html = BeautifulSoup(req.content, "html.parser")
+
+    # I dont really know what I am doing here...
+    # But seems to be working.
+    # The goal is to extract the main text from the URL.
+    texts = []
```

Comment:
i really dont know my way around with html parsing and such.. so this part of the code is surely of low qualitty...